spring-cloud-netflix-turbine/src/main/java/org/springframework/cloud/netflix/turbine/TurbineLifecycle.java

```diff
@@ -29,11 +30,13 @@
 public class TurbineLifecycle implements SmartLifecycle, Ordered {
 
 	private final InstanceDiscovery instanceDiscovery;
+	private final ClusterMonitorFactory<?> factory;
 
-	private boolean running;
+	private volatile boolean running;
```

Comment:
Possibly, though I doubt you'd see it in practice. I've used `AtomicBoolean` too.